components/trello/trello.app.js

```diff
@@ -70,35 +249,230 @@ module.exports = {
         key: this.$auth.oauth_access_token,
         secret: this.$auth.oauth_refresh_token,
       };
-      return (
-        await axios({
-          method: "POST",
-          url: this.$auth.oauth_signer_uri,
-          data: {
-            requestData,
-            token,
-          },
-        })
-      ).data;
-    },
-    async _makeRequest(config) {
-      const authorization = await this._getAuthorizationHeader(config);
+      return axios($ ?? this, {
+        method: "POST",
+        url: this.$auth.oauth_signer_uri,
+        data: {
+          requestData,
+          token,
+        },
+      });
+    },
+    async _makeRequest(config, $) {
+      const authorization = await this._getAuthorizationHeader(config, $);
       config.headers = {
         ...config.headers,
         authorization,
       };
       try {
-        return await axios(config);
+        return await axios($ ?? this, config);
       } catch (err) {
         console.log(err);
       }
     },
+    /**
+     * Archives a card.
+     *
+     * @param {string} idCard - the ID of the Card to archive.
+     * @returns an updated card object with `closed` (archived) property set to true.
+     * See more at the API docs:
+     * https://developer.atlassian.com/cloud/trello/rest/api-group-cards/#api-cards-id-put
+     */
+    async archiveCard(idCard, $) {
+      const config = {
+        url: `${this._getBaseUrl()}/cards/${idCard}`,
```

Comment:
Should this be ``url: `${this._getBaseUrl()}cards/${idCard}`,`` since `this._getBaseUrl()` ends with a `/`?

Also applies to some other methods in this file.